### ISSUE TITLE
Limit server life cycle management for public projects to project members

### ISSUE THREAD

**Issue by jgwerner** [3Blades/3blades#159]
Public projects allow for public access to project files and resources, including servers. 

Only collaborators should have access to start, stop and terminate servers. We should consider a new role / policy for these members, such as `workspaces managers`.

**Comment by johngriebel** [comment]
@jgwerner I don't think an entire new role is necessary, we recently added a new permission `write_server`, which I think would be appropriate for this purpose.

**Comment by jgwerner** [comment]
@johngriebel great. So we just need to make sure we can surface that option in the UI somehow.

**Comment by johngriebel** [comment]
Yup. It was implemented with teams in mind, but we can easily use it elsewhere of course. We will make it part of the design meeting Monday.

**Comment by raygesualdo** [comment]
@johngriebel Are the permissions surfaced through the API at the moment? If not, we could create an endpoint that lists the currently logged in user's permissions for a specific project.
